extensions/amp-story-dev-tools/0.1/amp-story-dev-tools-tab-logs.js

```diff
@@ -221,4 +194,66 @@ export class AmpStoryDevToolsTabLogs extends AMP.BaseElement {
     }
     this.mutateElement(() => logsTabSelector.appendChild(statusIcon));
   }
+
+  /**
+   * @private
+   * @return {!Element}
+   */
+  createSuccessMessage_() {
+    const logsContainer = this.element.ownerDocument.createElement('div');
+    logsContainer.appendChild(this.buildLogsTitle_(this.errorList_.length));
```

Comment:
Mentioned in comment above:
This could be moved to `buildLogsContent_` since it's used in `createErrorsList_` and `createSuccessMessage_`.